src/core/readpdb.cpp

```diff
@@ -41,7 +41,7 @@ static void add_lj_internal(std::set<PdbParser::itp_atomtype, PdbParser::itp_ato
 	continue;
       if(only_diagonal && (it->espresso_id != jt->espresso_id))
 	continue;
-      const double epsilon_ij = sqrt(it->epsilon * jt->epsilon);
+      const double epsilon_ij = sqrtf(it->epsilon * jt->epsilon);
```

Comment:
`std::sqrt`?

Reply:
Same thing. I just applied the fixes that Clang suggested.